### ISSUE TITLE
todo: link to mailing list

### ISSUE THREAD

**Issue by rgommers** [mingwpy/mingwpy.github.io#17]
which is at https://groups.google.com/forum/#!forum/mingwpy

**Comment by carlkl** [comment]
I configured the ML as a forum, is this ok?

**Comment by rgommers** [comment]
Yes, it looks fine to me. I could choose the normal mailing list options about receiving every email, one summary per day, etc.

I'll send a message to it now to be sure.

**Comment by rgommers** [comment]
Yep, works fine. Email address is mingwpy@googlegroups.com

**Comment by matthew-brett** [comment]
Is it possible to join the mailing list in an automated way?  I "applied"
to join just now, but I assume you (Carl) have to agree?

**Comment by carlkl** [comment]
I changed your status (@matthew-brett,@rgommers) to manager and opened the ML for all. If this ML gets misused I have to revert the permissions to post.

**Comment by carlkl** [comment]
@rgommers, maybe it's to early to ask this question: is there any opinion about this:
should numfocus take over http://mingwpy.org?

**Comment by rgommers** [comment]
> @rgommers, maybe it's to early to ask this question: is there any opinion about this:
> should numfocus take over http://mingwpy.org?

I'd say that in principle yes, this is a good idea. And not too early. However, we should configure it first so that mingwpy.org shows the same as what mingwpy.github.io now shows. Should be easy to do. Then the domain can be transferred I'd think.
